ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/converter/RaoResultExporter.java

```diff
@@ -1,17 +1,19 @@
 /*
- * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-package com.farao_community.farao.search_tree_rao;
+package com.farao_community.farao.rao_api.converter;
+
+import com.farao_community.farao.rao_api.RaoResult;
 
 import java.io.OutputStream;
 
 /**
- * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
+ * @author Philippe Edwards {@literal <philippe.edwards at rte-france.com>}
  */
-public interface SearchTreeRaoResultExporter {
+public interface RaoResultExporter {
```

Comment:
I don't know why we made this, but I don't think we really need an interface for RaoResult exporters.

It's quite heavy for a "simple" object, which have for now only a Json exporter.

Reply:
Yeah it was in case we want to export to a different format.

I guess we can remove it for now and add it back if we ever need a different export format!